modules/lti/src/main/resources/tools/courses/js/lti.recordings.js

```diff
@@ -75,6 +75,50 @@ function xhr(params, cb, fail, always) {
   request.send(params.data);
 }
 
+var user = [],
+    userURL = '/info/me.json';
+
+xhr({url: userURL, responseType: 'json'}, 
+    function(response) {
+        user = response.user;
+    }
+);
+
+$('#feedbackBtn').on('click', function() {
+  var parts = [],
+      feedbackUrl = "https://docs.google.com/forms/d/e/1FAIpQLSeXmOzYY3rwuB3Plj27kMcI-8B7PpBHkZOo_zi8dd15Zv3u4Q/viewform"; 
+
+  var tempVars = {
+    user: user,
+    mediaPackage_title: information.dcTitle,
+    mediaPackage_eventid: information.id,
+    mediaPackage_eventid: 'E['+information.id+']',
```

Comment:
This button is on the page listing all the episodes, so we don't have an event id surely? Only a series id from the LTI tool configuration.